integrations/standalone/tests/pageobjects/Composite.ts

```diff
@@ -28,6 +29,10 @@ export abstract class Composite {
     return new Select(this.page, this.locator, label);
   }
 
+  combobox(label: string) {

```

Comment:
There is already a select widget. What is the difference between combo and select?

Reply:
yes :)